### ISSUE TITLE
Unexpected token, Menu at 11:1

### ISSUE THREAD

**Issue by foreggs** [jamesmfriedman/rmwc#21]
./node_modules/rmwc/Menu/index.js
Module parse failed: Unexpected token (11:1)
You may need an appropriate loader to handle this file type.

Using the default CRA, error occurs when importing e.g. a Toolbar, but it seems to appear whenever i import anything from rmrc.

**Comment by foreggs** [comment]
Webpack seems to need the file to be using es5, I fixed the problem by importing from 'lib/rmwc' like so
```js
import {Button} from 'rmwc/lib/rmwc';
```
But this doesn't seem right at all, can you make the `index.js` contain 'lib/rmwc'?

**Comment by jamesmfriedman** [comment]
Sorry about that @foreggs, I need to add some documentation around the behavior. This is related to #20 

TLDR: include node_modules/rmwc with your webpack babel-loeader (node_modules is usually ignored)

- Currently, the package is published to NPM with ES6 code
- Google's MDC required that the components be structured in a way that can be consumed al-a-carte so it's setup to import the ES6 components directly
- The lib version is a UMD compiled bundle, so it's babelfied and compressed down to ES5 so it's safe for consumption like you're doing
- If you'd like to import the way that you're referencing, you just need to add `nodule_modules/rmwc` to the babel-loaders include.
- This is hot on my list of items to fix, just a minor annoyance right now.

**Comment by jamesmfriedman** [comment]
@foreggs Ah, just caught that you're using CRA, extra annoying, if you haven't run eject you'll need to use something like `react-app-rewired` to modify the babel config. I'll see about getting this fixed in the next few days. Thanks for trying out the project, sorry for the hiccup.

**Comment by jamesmfriedman** [comment]
@foreggs, The issue is resolved in 0.0.1-beta7. After you update it should all work as expected, no extra configuration required.
